### PR TITLE
Cleanup the Homepage

### PR DESCRIPTION
**Pull request by davetcoleman**
Cleanup homepage, replace image, move attribution

image

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -31,4 +28,15 @@ Table Of Contents
    doc/concepts/concepts
    doc/how_to_contribute/how_to_contribute
    doc/examples/examples
-   doc/attribution/attribution
+
+Attribution
+-----------
+
+Some major past contributors to the MoveIt tutorials are listed in chronological order: Sachin Chitta, Dave Hershberger, Acorn Pooley, Dave Coleman, Michael Gorner, Francisco Suarez, Mike Lautman, Tyler Waver, David Lu!!, Vatan Tezer, and Andy Zelenak. These are just some of the `46+ Contributors over the years <https://github.com/ros-planning/moveit2_tutorials/graphs/contributors>`_ who have a big impact on this documentation.
```
Comment (jackcenter) [inline index.rst:35]:
```suggestion
Some major past contributors to the MoveIt tutorials are listed in chronological order: Sachin Chitta, Dave Hershberger, Acorn Pooley, Dave Coleman, Michael Gorner, Francisco Suarez, Mike Lautman, Tyler Weaver, David Lu!!, Vatan Tezer, and Andy Zelenak. These are just some of the `46+ Contributors over the years <https://github.com/ros-planning/moveit2_tutorials/graphs/contributors>`_ who have a big impact on this documentation.
```

File: index.rst [hunk 2]
```diff
@@ -31,4 +28,15 @@ Table Of Contents
...
+
+Help us improve these docs and we'll be happy to include you here also!
+
+**Corporate Sponsorship**
+
+* The tutorials had a major update in 2018 during a code sprint sponsored by Franka Emika in collaboration with PickNik (`Check out the blog post! <http://moveit.ros.org/moveit!/ros/2018/02/26/tutorials-documentation-codesprint.html>`_)
```
Comment (jackcenter) [inline index.rst:41]:
```suggestion
* The tutorials had a major update in 2018 during a code sprint sponsored by Franka Emika in collaboration with PickNik Robotics (`Check out the blog post! <http://moveit.ros.org/moveit!/ros/2018/02/26/tutorials-documentation-codesprint.html>`_)
```

File: index.rst [hunk 3]
```diff
@@ -1,15 +1,12 @@
 MoveIt 2 Documentation
 ======================
 
-MoveIt 2 is the robotics manipulation platform for ROS 2. For ROS 1, see `MoveIt 1 <https://ros-planning.github.io/moveit_tutorials>`_.
+Welcome to unified MoveIt documentation, which includes tutorials, how-to-guides, core concepts, and more.
 
-.. image:: doc/tutorials/quickstart_in_rviz/rviz_plugin_head.png
-   :width: 700px
+MoveIt 2 is the robotics manipulation platform for ROS 2, which had its first release in 2019. For ROS 1 documentation, see `MoveIt 1 tutorials <https://ros-planning.github.io/moveit_tutorials>`_.
```
Comment (tylerjw) [inline index.rst:6]: [nit] swap these so the text "MoveIt 2 is the robotics manipulation platform for ROS 2, which had its first release in 2019...." is what would show up in a google search.

Also, we could still use a much better description of what Moveit is and why someone would use it.  "the robotics manipulation platform" is really vague imo.

Comment (davetcoleman) [reply index.rst:6]: I think the important part for search results is that this is the unified documentation with X, Y, Z. I would like to keep the order as-is.

What MoveIt is, is less important here. That's more the topic of the MoveIt homepage. I have expanded the description of what MoveIt is per your suggestion.

**Review by tylerjw — APPROVED** [review]
lgtm... tested locally

**Review by vatanaksoytezer — APPROVED** [review]

**Pull request merged**